alchemy/src/cloudflare/images.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @see https://developers.cloudflare.com/images/transform-images/bindings/
+ */
+export class Images {
```

Comment:
maybe add docs showing how to configure it when creating a Worker and at runtime